### ISSUE TITLE
[EDIT] Subtitles dragged to timeline subtitle name should be use SRT file timecodes

### ISSUE THREAD

**Issue by pgilfernandez** [olive-editor/olive#1929]
**Commit Hash** 
912dcfd

**Platform** 
macOS 12.3 (probably all)

**Summary**
This issue is related with issue https://github.com/olive-editor/olive/issues/1928 because I think I didn't explain myself correctly there:
When I said "...without offset" I meant that subtitles should be placed using the SRT file timecodes, that is, if I have a 'example.srt' file:
```
1
00:00:01,000 --> 00:00:02,000
Hello

2
00:00:03,000 --> 00:00:04,000
Bye
````
So what I meant is that when dragging the example.srt asset from the Project to the subtitle track name the first subtitle should be placed at 00:00:01 (and disappear at 00:00:02) while second subtitle should be placed at 00:00:03 (and disappear at 00:00:04). That is:
<img width="489" alt="Captura de Pantalla 2022-05-13 a las 23 42 15" src="https://user-images.githubusercontent.com/5942369/168393436-f91e741c-cb9f-444d-9a8d-5449ac0db577.png">

While in the current implementation (not desired as this can reproduced dropping the subtitle asset in the timeline and leaving it at the start of it) the first one is placed at 00:00:00 (and disappear at 00:00:01) while second subtitle is placed at 00:00:02 (and disappear at 00:00:03). That is:
<img width="488" alt="Captura de Pantalla 2022-05-13 a las 23 41 25" src="https://user-images.githubusercontent.com/5942369/168393559-1c7469c8-3462-45c1-a121-9d8986069fb2.png">

I guess you will find it logical and I would like to apologize for my bad English or/and for not explaining myself correctly in the previous issue...

**Additional Information / Output**
BTW, I love how this features are getting shaped, I think people are going to love them =)

**Comment by itsmattkc** [comment]
Heh actually I didn't misunderstand you, this was just a logic bug that I didn't notice. Fixed in fb72ca4997e4ada5ca4eef7198668ff7f27a3f52

**Comment by pgilfernandez** [comment]
Oh, nice to hear, hehehe, thanks
